Testing/Data/DocTest/EMU00102347.nxs_v2.md5

```diff
@@ -0,0 +1 @@
+20855e63d0beb6aba0490e43f20297d6
```

Comment:
Thanks for changing the files. Can we avoid copying the same file to multiple directories? It's easy to only update one reference in the future and end up confused in some places. The Unit tests actually have access to all files in DocTest + UnitTest and the system tests have access to everything so from what I can see here if you just include it in `DocTest` all of the tests can access it?

Reply:
Sorry, wrongly assumed it had to be present in each directory. I've now deleted the unnecessary copies.

Reply:
No worries. We wanted to keep doc tests separate so we could just zip it up and make it available but everyone assumes as you did that they are completely separate. Maybe I should add a readme or make DocTest and subdirectory of UnitTest or something.